main.tf

```diff
@@ -124,6 +130,43 @@ resource "aws_elasticache_cluster" "memcache" {
   }
 }
 
+/*
+ * Create RDS root password
+ */
+resource "random_password" "db_root" {
+  count = var.session_store_type == "sql" ? 1 : 0
+
+  length = 16
+}
+
+/*
+ * Create RDS database for session store, if session_store_type is "sql"
+ */
+module "rds" {
```

Comment:
I'm a little confused.
If ssp-base has its own database, why do we need to create and RDS database for the IdP Hub?

Reply:
ssp-base is set up to *use* a database, but since the hub runs on its own, apart from idp-in-a-box, it needs its own database.

Reply:
I finally found it. It looks like silauth is what wants to use the ssp-base database.  I see there are migrations in silauth.  I haven't tracked down yet where/when they are run.